Childrens-Social-Care-CPD/Views/Shared/_VideoResource.cshtml

```diff
@@ -8,16 +8,52 @@
 
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-three-quarters">
-        <video id="videoPlayer" preload="none">
-            <source src="@Model.Video.File.Url">
-        </video>
-        <details class="govuk-details govuk-!-margin-top-3" data-module="govuk-details">
-            <summary class="govuk-details__summary"> <span class="govuk-details__summary-text">View transcript  </span></summary>
-            <div class="govuk-details__text">
-                @{
-                    await Html.RenderPartialAsync("_RichText", Model.Transcript);
-                }
-            </div>
-        </details>
+        @if(!string.IsNullOrEmpty(Model.EmbededTitle) && !string.IsNullOrEmpty(Model.EmbededSourceUrl))
+        {
+            <h2>
+                @Model.EmbededTitle
+            </h2>
+            if (@Model.EmbededSourceUrl.IndexOf("vimeo") > -1)
+            {
+                <iframe width="560" height="315"
+                        src="@Model.EmbededSourceUrl"
+                        title="@Model.EmbededTitle"
+                        allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+                        allowfullscreen>
+                </iframe>
+            }
+            else
+            {
+                <video id="SomeID" name="media" src="@Model.EmbededSourceUrl" >
+                    <track label="English" kind="captions" srclang="en" default>
+                </video>
+            }
+
+            if (Model.Transcript != null && Model.Transcript.Content != null && Model.Transcript.Content.Any())
```

Comment:
You can use the null conditional operator here, e.g. Model.Transcript?.Content != null

Reply:
Fixed